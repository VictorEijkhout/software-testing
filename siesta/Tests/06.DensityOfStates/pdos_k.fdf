%include ../base_fe.fdf
SystemLabel       pdos_k

PDOS.kgrid.MonkhorstPack [10 10 10]
%block ProjectedDensityOfStates
EF -5.00 5.00 0.200000 1000 eV
%endblock ProjectedDensityOfStates

