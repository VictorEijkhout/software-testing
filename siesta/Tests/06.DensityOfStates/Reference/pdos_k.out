Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/06.DensityOfStates/work/pdos_k
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:36:19

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.72772
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel       pdos_k
PDOS.kgrid.MonkhorstPack [10 10 10]
%block ProjectedDensityOfStates
EF -5.00 5.00 0.200000 1000 eV
%endblock ProjectedDensityOfStates
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: 
reinit: -----------------------------------------------------------------------
reinit: System Label: pdos_k
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   26 Label: Fe_soc


---- Processing specs for species: Fe_soc
Ground state valence configuration:   4s02  3d06
Reading pseudopotential information in PSML from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Fe_soc.psml
PSML file version: 1.1
Using libxc ids:  101 130
GGA--PBE XC_GGA_X_PBE--XC_GGA_C_PBE pb
PSML uuid: 89456970-44bf-11e8-4d23-a6a6edf19e30
Semicore shell(s) with  8 electrons included in the valence for Fe_soc

---- Pseudopotential check for Fe

Pseudopotential generated from a fully relativistic atomic calculation
There are spin-orbit semi-local pseudopotentials available

Pseudized shells:
3s( 2.00) rc: 1.26
3p( 6.00) rc: 1.21
3d( 6.00) rc: 1.57
Valence configuration for ps generation: 3s:3p:3d:4s: Total charge:  16.000000
Semicore shell(s): 3s 3p
Fallback to non-perturbative polarization scheme for Fe_soc
Please see the manual if this is not desired in principle
For Fe_soc, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.
Fe_soc pseudopotential only contains V_ls up to l=2 -- lmxkb reset.

<basis_specs>
===============================================================================
Fe_soc               Z=  26    Mass=  55.845        Charge= 0.17977+309
Lmxo=2 Lmxkb= 2    BasisType=split      Semic=T
L=0  Nsemic=1  Cnfigmx=4
          i=1  nzeta=1  polorb=0  (3s)  (semicore shell)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
          i=2  nzeta=1  polorb=0  (4s)  (to be polarized non-perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=1  Cnfigmx=4
          i=1  nzeta=1  polorb=0  (3p)  (semicore shell)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
          i=2  nzeta=1  polorb=0  (4p)  (non-perturbative polarization shell)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:   0.16956-319
               qwid:   0.11198-314
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=3
          i=1  nzeta=1  polorb=0  (3d)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
-------------------------------------------------------------------------------
L=0  Nkbl=2  erefs: 0.17977+309 0.17977+309
L=1  Nkbl=2  erefs: 0.17977+309 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Fe_soc                (Z =  26)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ONCVPSP-3.Hamann's oncvpsp                        
Valence charge in pseudo generation:   16.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: GGA Perdew, Burke & Ernzerhof 1996
Got Vlocal (oncv-fit) from psml data
Choosing vlocal chloc cutoff:  3.831373
qtot up to nchloc:   15.99793860
atom: Maximum radius for chlocal:    3.83137
atom: Maximum radius for r*vlocal+2*Zval:    3.83137
Reading KB projs from Fe_soc psml data
Using (derived) scalar-relativistic projectors in PSML file

PSML: Kleinman-Bylander projectors: 
   l= 0   rc=  1.280242   Ekb=  2.929588
   l= 0   rc=  1.280242   Ekb= 20.336184
   l= 1   rc=  1.221004   Ekb=  3.212733
   l= 1   rc=  1.221004   Ekb= -0.970660
   l= 1   rc=  1.221004   Ekb=  0.001568
   l= 1   rc=  1.221004   Ekb= -0.000083
   l= 2   rc=  1.586526   Ekb=-11.013454
   l= 2   rc=  1.586526   Ekb= -4.065821

PSML: Total number of  Kleinman-Bylander projectors:  24
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 3s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    2.378892
                 energy =   -6.894702
                kinetic =    3.214823
    potential(screened) =  -10.109525
       potential(ionic) =  -37.547812

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.575234
                 energy =   -0.437248
                kinetic =    1.185569
    potential(screened) =   -1.622817
       potential(ionic) =  -14.647486

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 3p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    2.635420
                 energy =   -4.398765
                kinetic =    7.852272
    potential(screened) =  -12.251038
       potential(ionic) =  -38.913597

SPLIT: Basis orbitals for state 4p
SPLIT: Non-perturbative polarization orbs for state 4s
SPLIT: rc(1st zeta) (re)set to: 6.5752 (expansion factor: 1.0000)
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.575234
                 energy =   -0.021587
                kinetic =    1.092964
    potential(screened) =   -1.114551
       potential(ionic) =  -11.623296

SPLIT: Orbitals with angular momentum L= 2

SPLIT: Basis orbitals for state 3d

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.010000 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.097563
                 energy =   -0.539341
                kinetic =   10.289951
    potential(screened) =  -10.829292
       potential(ionic) =  -34.384996
atom: Total number of Sankey-type orbitals: 13

atm_pop: Valence configuration (for local Pseudopot. screening):
 3s( 2.00) 4s( 2.00)                                                  
 3p( 6.00) 4p( 0.00)                                                  
 3d( 6.00)                                                            
Vna: chval, zval:   16.00000  16.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.575234
comcore: Pseudo-core radius Rcore=  1.729904

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   26 Fe_soc                  # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Fe_soc                5                    # Species label, number of l-shells
 n=3   0   1                         # n, l, Nzeta 
   2.379   
   1.000   
 n=4   0   1                         # n, l, Nzeta 
   6.575   
   1.000   
 n=3   1   1                         # n, l, Nzeta 
   2.635   
   1.000   
 n=4   1   1                         # n, l, Nzeta 
   6.575   
   1.000   
 n=3   2   1                         # n, l, Nzeta 
   4.098   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

CH_OVERLAP: New q2cut (Ry) =   3100.00
CH_OVERLAP: New q2cut (Ry) =   3844.00
CH_OVERLAP: New q2cut (Ry) =   4766.56
CH_OVERLAP: New q2cut (Ry) =   5910.53
Dumping basis to NetCDF file Fe_soc.ion.nc
coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      1    13    24

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   300.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =       50
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   116.0452 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 10
mix.SCF:    Linear mixing weight                 =     0.010000
mix.SCF:    Mixing weight                        =     0.010000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.0100
  weight.linear 0.0100
  history 10
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:    16.000000
Total ionic charge:    16.000000
 Kpoints in:          196 . Kpoints trimmed:          172

siesta: k-grid: Number of k-points = 172
siesta: k-points from Monkhorst-Pack grid
siesta: k-cutoff (effective) =     8.438 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     7     0     0      0.000
siesta: k-grid:     0     7     0      0.000
siesta: k-grid:     0     0     7      0.000
k-point displ. along   1 input, could be:     0.00    0.50
k-point displ. along   2 input, could be:     0.00    0.50
k-point displ. along   3 input, could be:     0.00    0.50
 Kpoints in:          600 . Kpoints trimmed:          560

siesta: PDOS k-grid: Number of k-points = 560
siesta: PDOS k-points from Monkhorst-Pack grid
siesta: PDOS k-cutoff (effective) =    12.054 Ang
siesta: PDOS k-point supercell and displacements
siesta: k-grid:    10     0     0      0.000
siesta: k-grid:     0    10     0      0.000
siesta: k-grid:     0     0    10      0.000

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000

superc: Internal auxiliary supercell:     9 x     9 x     9  =     729
superc: Number of atoms, orbitals, and projectors:    729   9477  17496


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     9 x     9 x     9  =     729
superc: Number of atoms, orbitals, and projectors:    729   9477  17496

outcell: Unit cell vectors (Ang):
       -1.391873    1.391873    1.391873
        1.391873   -1.391873    1.391873
        1.391873    1.391873   -1.391873

outcell: Cell vector modules (Ang)   :    2.410795    2.410795    2.410795
outcell: Cell angles (23,13,12) (deg):    109.4712    109.4712    109.4712
outcell: Cell volume (Ang**3)        :     10.7860
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=13 nrows=13 sparsity=65.9822 nnzs=11151, refcount: 7>
  <dData1D:(new from dSpData1D) n=11151, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=13 nrows=13 sparsity=65.9822 nnzs=11151, refcount: 8>
  <dData2D:DM n=11151 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      27     187

InitMesh: MESH = 24 x 24 x 24 = 13824
InitMesh: Mesh cutoff (required, used) =   300.000   410.861 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -612.666314
siesta: Eions   =      4922.283762
siesta: Ena     =       584.711018
siesta: Ekin    =      1533.576536
siesta: Enl     =      -180.926584
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        53.854728
siesta: DUscf   =         1.868169
siesta: DUext   =         0.000000
siesta: Ex      =      -451.588668
siesta: Ec      =       -26.534794
siesta: Exc     =      -478.123462
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -3401.683609
siesta: Etot    =     -3407.323357
siesta: FreeEng =     -3407.324548

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1    -3401.683609    -3407.323357    -3407.324548  1.788244 -1.973391  6.534145
timer: Routine,Calls,Time,% = IterSCF        1       0.126  13.44
   scf:    2    -3407.382934    -3407.353221    -3407.354416  0.005677 -2.017324  6.405489
   scf:    3    -3408.403185    -3408.091646    -3408.093368  0.243185 -4.179763  0.218007
   scf:    4    -3408.093607    -3408.092686    -3408.094400  0.003198 -4.147044  0.148479
   scf:    5    -3408.094390    -3408.093883    -3408.095728  0.007639 -4.068711  0.051592
   scf:    6    -3408.093916    -3408.093900    -3408.095744  0.000266 -4.071442  0.043860
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 5
   scf:    7    -3408.093941    -3408.093932    -3408.095779  0.001449 -4.086240  0.002085
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 6
   scf:    8    -3408.093933    -3408.093933    -3408.095779  0.000054 -4.085714  0.002142
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 7
   scf:    9    -3408.093932    -3408.093932    -3408.095779  0.000001 -4.085747  0.002118
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 8
   scf:   10    -3408.093932    -3408.093932    -3408.095779  0.000001 -4.085759  0.002097
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 9
   scf:   11    -3408.093932    -3408.093932    -3408.095779  0.000001 -4.085772  0.002076
mix: Pulay -- inversion failed, > SVD [rank/size] 2 / 9
   scf:   12    -3408.093932    -3408.093932    -3408.095779  0.000001 -4.085785  0.002056
mix: Pulay -- inversion failed, > SVD [rank/size] 3 / 9
   scf:   13    -3408.093921    -3408.093927    -3408.095779  0.000113 -4.087859  0.002307
mix: Pulay -- inversion failed, > SVD [rank/size] 3 / 9
   scf:   14    -3408.093928    -3408.093927    -3408.095779  0.000069 -4.087053  0.000120

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000689906
max |H_out - H_in|      (eV) :     0.0001201322
SCF cycle converged after 14 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      27     187

siesta: E_KS(eV) =            -3408.0939

siesta: E_KS - E_eggbox =     -3408.0939

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000000    0.000000   -0.000000
----------------------------------------
   Max    0.000000
   Res    0.000000    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.000000    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):        1.68        1.68        1.68        0.31        0.31        0.31
(Free)E + p*V (eV/cell)    -3408.1071
Target enthalpy (eV/cell)    -3408.0958

siesta: PDOS info
siesta: Shifting energies with respect to Fermi-level
siesta: E1 -- E2, sigma [eV], nhist:   -9.087 --    0.913    0.200 1000

siesta: Program's energy decomposition (eV):
siesta: Ebs     =      -576.262640
siesta: Eions   =      4922.283762
siesta: Ena     =       584.711018
siesta: Ekin    =      1576.956442
siesta: Enl     =      -192.634158
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =        28.764607
siesta: DUscf   =         0.506321
siesta: DUext   =         0.000000
siesta: Ex      =      -457.590152
siesta: Ec      =       -26.524244
siesta: Exc     =      -484.114396
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =     -3408.093928
siesta: Etot    =     -3408.093927
siesta: FreeEng =     -3408.095779

siesta: Final energy (eV):
siesta:  Band Struct. =    -576.262640
siesta:       Kinetic =    1576.956442
siesta:       Hartree =     818.677355
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -457.590152
siesta:         Corr. =     -26.524244
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -484.114396
siesta:  Ion-electron =   -3304.504715
siesta:       Ion-ion =   -2015.108614
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =   -3408.093927
siesta:         Fermi =      -4.087053

siesta: Stress tensor (static) (eV/Ang**3):
siesta:     0.001046    0.000191    0.000191
siesta:     0.000191    0.001046    0.000191
siesta:     0.000191    0.000191    0.001046

siesta: Cell volume =         10.785960 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:          -0.00001139         -0.00001139  Ry/Bohr**3
siesta:          -0.00104568         -0.00104568  eV/Ang**3
siesta:          -1.67535774         -1.67535774  kBar
(Free)E+ p_basis*V_orbitals  =       -3407.577328
(Free)Eharris+ p_basis*V_orbitals  =       -3407.577329

cite: Please see "pdos_k.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       3.042
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       3.028       3.028   100.00
Setup                  1       0.071       0.071     2.35
bands                  1       0.000       0.000     0.00
KSV_init               1       0.000       0.000     0.00
IterGeom               1       2.781       2.781    91.86
geom_init              1       0.738       0.738    24.37
state_init             1       0.152       0.152     5.01
hsparse                1       0.002       0.002     0.07
overlap                1       0.053       0.053     1.74
Setup_H0               1       0.528       0.528    17.45
naefs                  2       0.000       0.000     0.01
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.026       0.052     1.71
nlefsm                 2       0.176       0.352    11.64
DHSCF_Init             1       0.157       0.157     5.18
DHSCF1                 1       0.022       0.022     0.74
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.134       0.134     4.44
REMESH                 1       0.026       0.026     0.86
REORD                 47       0.000       0.001     0.02
PHION                  1       0.093       0.093     3.06
COMM_BSC              67       0.000       0.001     0.02
POISON                17       0.000       0.004     0.14
fft                   34       0.000       0.003     0.08
setup_H               15       0.059       0.882    29.12
DHSCF                 16       0.066       1.063    35.12
DHSCF3                16       0.059       0.937    30.95
rhoofd                16       0.030       0.480    15.85
XC                    16       0.006       0.102     3.35
GXC-CellXC            16       0.006       0.101     3.34
gridxc@cellXC         16       0.006       0.101     3.34
vmat                  16       0.022       0.350    11.55
IterSCF               14       0.130       1.826    60.30
compute_dm            14       0.071       0.996    32.89
diagon                14       0.071       0.995    32.87
c-eigval            2408       0.000       0.308    10.19
c-buildHS           2408       0.000       0.278     9.19
cdiag               5376       0.000       0.095     3.13
cdiag1              5376       0.000       0.005     0.15
cdiag2              5376       0.000       0.014     0.47
cdiag3              5376       0.000       0.058     1.92
c-eigvec            2408       0.000       0.323    10.66
cdiag4              2968       0.000       0.003     0.11
c-buildD            2408       0.000       0.358    11.82
MIXER                 13       0.000       0.004     0.12
PostSCF                1       0.217       0.217     7.18
DHSCF4                 1       0.126       0.126     4.17
dfscf                  1       0.109       0.109     3.61
overfsm                1       0.002       0.002     0.05
state_analysis         1       0.000       0.000     0.00
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.175       0.175     5.78
pdos                   1       0.174       0.174     5.75
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:36:22
Job completed
